File: fizzbuzz.pl
$number = 0;
$finish = 20;

while ($number < $finish) {
  $number += 1;

  if (($number % 3 == 0) && ($number % 5 == 0)) {
    print "FizzBuzz!\n";
  } elsif ($number % 3 == 0) {
    print "Fizz!\n";
  } elsif ($number % 5 == 0) {
    print "Buzz!\n";
  } else {
    print "$number\n";
  }
}

File: fizzbuzz.lua
for number = 1, 1000 do
    if (number % 15 == 0) then
        print("FizzBuzz!")
    elseif (number % 3 == 0) then
        print("Fizz!")
    elseif (number % 5 == 0) then
        print("Buzz!")
    else
        print(number)
    end
end
